### DISCUSSION TITLE
DPMD Running Error

### DISCUSSION THREAD

**Discussion by Taixi-Feng** [deepmodeling/deepmd-kit#4044] [Q&A]
I used the DEEPMD-LAMMPS for MD simulation, but met this error. Please tell me how to solve it, thanks.
ensorflow_cc.so;/root/deepmd-kit/lib/libtensorflow_framework.so
Not found: Op type not registered 'ProdEnvMatA' in binary running on localhost.localdomain. Make sure the Op and Kernel are registered in the binary running in this process. Note that if you are loading a saved graph which used ops from tf.contrib, accessing (e.g.) `tf.contrib.resampler` should be done before importing the graph, as contrib ops are lazily registered when the module is first accessed.
![image](https://github.com/user-attachments/assets/035d11ed-596c-4afe-b7d6-4ffd2550e945)

**Comment by njzjz** [comment]
Please post the entire error message and log.
